decentraland/screens/MyAssetsScreen.js

```diff
@@ -30,15 +30,33 @@ export class MyAssetsScreen extends React.Component {
       const shouldUpdate = !listsAreEqual(assetsFromBlockchain, boughtAssets);
       if (shouldUpdate) {
         // TODO: Add some UI indication that something unexpected happened
-        logWarn(
-          `Account's assets from blockchain aren't the same as the ones stored on the app.`
-        );
+        this._logAssetsInconsistency(assetsFromBlockchain, boughtAssets);
         removeFromMyAssetsList(boughtAssets);
         appendToMyAssetsList(assetsFromBlockchain);
       }
     }
   };
 
+  _logAssetsInconsistency = (fromBlockchain, fromState) => {
+    const fromBlockchainIds = fromBlockchain.map(asset => asset.id);
+    const fromStateIds = fromState.map(asset => asset.id);
+
+    const addedIds = fromBlockchainIds.filter(id => !fromStateIds.includes(id));
+    const removedIds = fromStateIds.filter(
+      id => !fromBlockchainIds.includes(id)
+    );
+
+    logWarn(
+      `Account's assets from blockchain aren't the same as the ones stored on the app.`
+    );
+
+    if (addedIds.length > 0)
+      logWarn(`Some assets added to MyLandScreen. IDs: [${addedIds}]`);
```

Comment:
This could probably be a `logInfo` rather than a `logWarn`. Either that or promote the removal one up from `logWarn` to `logError`

Reply:
Done.

Reply:
Awesome, thanks. One remaining thing on this thread over here:

https://github.com/tasitlabs/tasit/pull/333/commits/edb45201c90a443512e3e92005ba9b63bad15494#r277888035

Reply:
Okay, I'll resolve this one since the other thread is tracking it.

Reply:
Good call. Thanks.